common/app/model/EmailAddons.scala

```diff
@@ -226,6 +226,21 @@ case object OpinionUs extends FrontEmailMetadata {
   override val banner = Some("opinion.png")
 }
 
+case object ThisLandIsYourLand extends FrontEmailMetadata {
+  val name = "This Land Is Your Land"
+  override val banner = Some("this-land-is.png")
+}
+
+case object WinterOlympics2018 extends FrontEmailMetadata {
+  val name = "Winter Olympics 2018"
+  override val banner = Some("winter-olympics-2018.png")
+}
+
+case object TheUpsideWeeklyReport extends FrontEmailMetadata {
```

Comment:
Since this is being added to `articleEmails`, I think this needs to extend `ArticleEmailMetadata` rather than `FrontEmailMetadata`. We would therefore also need to define a `test`.